A3A/addons/core/functions/CREATE/fn_invaderPunish.sqf

```diff
@@ -30,7 +30,7 @@ params ["_mrkDest", "_mrkOrigin", "_delay"];
 ServerInfo_2("Launching CSAT Punishment Against %1 from %2", _mrkDest, _mrkOrigin);
 
 // Mostly to prevent fast travel
-bigAttackInProgress = true; publicVariable "bigAttackInProgress";
+//bigAttackInProgress = true; publicVariable "bigAttackInProgress";
```

Comment:
Surely a punishment is a big attack?

Reply:
This comment should be removed given it dictates if extra missions are created during already laggy events